### DISCUSSION TITLE
Scientific notations in Prometheus metrics

### DISCUSSION THREAD

**Discussion by AndreyPoleshuk** [roadrunner-server/roadrunner#1057] [Q&A] [answered]
Prometheus metrics with gauge type and value bigger than 100000 are exposed in the scientific notation like 1e+06. Is it possible somehow to manage it and expose it in simple number format as it is?

**Comment by rustatian** [comment, marked as answer]
Hey @AndreyPoleshuk. Unfortunately, we can't manage this format. Prometheus uses the `float` data types to represent the data, so, we send regular float numbers and Prometheus parses them on its own. IIRC, we can't change that.
